dask/array/core.py

```diff
@@ -286,6 +286,7 @@ def dotmany(A, B, leftfunc=None, rightfunc=None, **kwargs):
         A = map(leftfunc, A)
     if rightfunc:
         B = map(rightfunc, B)
+    breakpoint()
```

Comment:
I suspect this is left over from developing locally 

```suggestion
```

Reply:
d'oh!  Thanks @jrbourbeau !